Simplified/Reader2/Internal/LibraryModule.swift

```diff
@@ -15,28 +15,67 @@ import R2Shared
 import R2Streamer
 import UIKit
 
-// TODO: SIMPLY-2656 Review "module" nomenclature
-
 /// The Library module handles the presentation of the bookshelf, and the publications' management.
 protocol LibraryModuleAPI {
-  var libraryService: LibraryService {get}
   
-  /// Loads the R2 DRM object for the given publication.
-  func loadDRM(for book: NYPLBook, completion: @escaping (CancellableResult<DRM?>) -> Void)
+  var delegate: LibraryModuleDelegate? { get }
+  
+  /// Root navigation controller containing the Library.
```

Comment:
not your fault, but what does Library mean here? maybe the SimplyE equivalent is actually "my books + reservations"? Or just downloaded books?

Reply:
`LibraryModule` is not needed in our app (it shows *library*, books), but was used in a protocol. Replaced with LibraryService we actually need from that module.